dotnet/src/SemanticKernel.Core/Functions/SKFunctionFromMethod.cs

```diff
@@ -78,16 +78,7 @@ internal sealed class SKFunctionFromMethod : ISKFunction
     }
 
     /// <inheritdoc/>
-    public string Name { get; }
-
-    /// <inheritdoc/>
-    public string Description { get; }
-
-    /// <inheritdoc/>
-    IEnumerable<AIRequestSettings> ISKFunction.ModelSettings => Enumerable.Empty<AIRequestSettings>();
-
-    /// <inheritdoc/>
-    public SKFunctionMetadata GetMetadata() =>
+    public override SKFunctionMetadata GetMetadata() =>
         this._view ??=
```

Comment:
nit: not related to your changes. The `_view` field can be renamed to `_metadata`.

Reply:
Tracking this work here: https://github.com/microsoft/semantic-kernel/issues/3605